Extensions/outbox.js

```diff
@@ -475,7 +475,10 @@ XKit.extensions.outbox = new Object({
 			m_html = m_html + XKit.extensions.outbox.render(m_obj, m_message);
 
 		}
-
+		if ($('.no_posts_found').length > 0) {
+			$('.no_posts_found').after("<div id='posts' class='posts'></div>");
```

Comment:
I believe the DOM element that wraps the outbox elements is actually `<ol id="posts" class="posts post-avatar--static">`. I don't know if it matters, but the current implementation wraps the children `li` elements with a `div`, which seems improper.